zpm.c

```diff
@@ -394,18 +417,25 @@ int plugin_print_list() {
     char* plugin_name = strtok(listing, "\n");
 
     if (!strcmp(listing, "Nothing to show.")) {
-        printf("%s\n", listing);
+        zpm_msg(ZPM_MSG, "%s\n", listing);
         free(listing);
         return 1;
     }
     while (plugin_name) {
        char* hash = plugin_get_hash(plugin_name);
-       printf("%s", plugin_name);
        if (hash) {
-           printf("@%s", hash);
+           if(zpm_has_color) {
+               zpm_msg(ZPM_NONE, "\033[1m%s\033[0m@%s", plugin_name, hash);
+           } else {
+               zpm_msg(ZPM_NONE, "%s@%s", plugin_name, hash);
+           }
            free(hash);
        } else {
-          printf("\n");
+           if(zpm_has_color) {
```

Comment:
* because **bold** relates to *fancy* output.
  * because **bold** messes with cram 😕